docs/code/IDataViewTypeSystem.md

```diff
@@ -383,82 +383,53 @@ Key types are instances of the sealed class `KeyType`, which derives from
 
 In addition to its underlying type, a key type specifies:
 
-* A count value, between `0` and `int.MaxValue`, inclusive
+* A count value, between `1` and `ulong.MaxValue`, inclusive
 
-* A "minimum" value, between `0` and `ulong.MaxValue`, inclusive
-
-* A Boolean value indicating whether the values of the key type are contiguous
-
-Regardless of the minimum and count values, the representational value zero
+Regardless of the count value, the representational value zero
 always means `NA` and the representational value one is always the first valid
 value of the key type.
 
 Notes:
 
-* The `Count` property returns the count of the key type. This is of type
-  `int`, but is required to be non-negative. When `Count` is zero, the key
-  type has no known or useful maximum value. Otherwise, the legal
-  representation values are from one up to and including `Count`. The `Count`
-  is required to be representable in the underlying type, so, for example, the
-  `Count` value of a key type based on `System.Byte` must not exceed `255`. As
-  an example of the usefulness of the `Count` property, consider the
-  `KeyToVector` transform implemented as part of ML.NET. It maps from a key
-  type value to an indicator vector. The length of the vector is the `Count`
-  of the key type, which is required to be positive. For a key value of `k`,
-  with `1 ≤ k ≤ Count`, the resulting vector has a value of one in the
-  (`k-1`)th slot, and zero in all other slots. An `NA` value (with
-  representation zero) is mapped to the all- zero vector of length `Count`.
+* The `Count` property returns the count, or cardinality of the key type. This 
+  is of type `ulong`. The legal representation values are from one up to 
+  and including `Count`. The `Count` is required to be representable in the 
+  underlying type, so, for example, the `Count` value of a key type based
+  on `System.Byte` must not exceed `255`. As an example of the usefulness 
+  of the `Count` property, consider the `KeyToVector` transform implemented
+  as part of ML.NET. It maps from a key type value to an indicator vector.
+  The length of the vector is the `Count` of the key type, which is required
+  to be positive. For a key value of `k`, with `1 ≤ k ≤ Count`, the resulting
+  vector has a value of one in the (`k-1`)th slot, and zero in all other slots.
```

Comment:
Maybe
```suggestion
  vector has a value of one in the (`k`)th slot, and zero in all other slots.
```
? The 1st slot is indexed by key value 1, isn't it?

Reply:
Depends on whether for teh sake of this discussion we're considering the "first" slot to be literally 1st or 0th.

Reply:
I see. `k-1` is fine, so let me close this comment.